docs/self-signed-ssl.md

```diff
@@ -5,3 +5,18 @@ When developing locally, by default we are simulating the authentication workflo
 A local self-signed certificate can be created either using OS built-in tools, or using tools such as `mkcert (https://github.com/FiloSottile/mkcert)`. This tool will create a certificate for the ip address(es) or domains passed as input, e.g.: `mkcert localhost 127.0.0.1`.
 
 Once a certificate is created, it can be put in the root of the solution, and referenced when configuring the site over HTTPS.
+
+#### Windows steps
+
+1. Navigate to back-office/appeals/web/docs in your terminal
```

Comment:
Is this correct? Maybe should clarify that you'll create the certificate in the folder from where you run the command, and the docker image should be created from the same folder in order to map the certs correctly to the docker fs.